Documentation/Storage-Configuration/Advanced/ceph-osd-mgmt.md

```diff
@@ -121,7 +138,7 @@ In the osd-purge.yaml, change the `<OSD-IDs>` to the ID(s) of the OSDs you want
 2. When the job is completed, review the logs to ensure success: `kubectl -n rook-ceph logs -l app=rook-ceph-purge-osd`
 3. When finished, you can delete the job: `kubectl delete -f osd-purge.yaml`
 
-If you want to remove OSDs by hand, continue with the following sections. However, we recommend you to use the above-mentioned job to avoid operation errors.
+If you want to remove OSDs by hand, continue with the following sections. However, we recommend you to use the above-mentioneds steps to avoid operation errors.
```

Comment:
```suggestion
If you want to remove OSDs by hand, continue with the following sections. However, we recommend you to use the above-mentioned steps to avoid operation errors.
```